tests/zero_code_change/horovod/tensorflow/utils.py

```diff
@@ -0,0 +1,71 @@
+# Standard Library
+import json
+import subprocess
+import sys
+from pathlib import Path
+
+# Third Party
+from tensorflow.python.client import device_lib
+
+
+def get_available_gpus():
+    local_device_protos = device_lib.list_local_devices()
+    return [x.name for x in local_device_protos if x.device_type == "GPU"]
+
+
+def build_json(
+    out_dir="/home/ubuntu/smdtensors",
```

Comment:
Would prefer to place test artifacts in `/tmp`.

Reply:
Will change

Reply:
This wasn't being used, as out_dir was always passed from the fixture. Made it required param